### DISCUSSION TITLE
Organograma

### DISCUSSION THREAD

**Discussion by enioluis** [freeplane/freeplane#458] [Help]
bom dia,
alguem poderia me ajudar em Como fazer um organograma na ferramenta?

**Comment by euu2021** [comment]
Você quer um organograma vertical (ou seja, com os nós sendo posicionados abaixo do nó central)? 

Se sim, então eu recomendo que você use nós flutuantes e os conecte, assim:

- inserir > novo nó > novo nó flutuante
- posicione o nó livremente
- para conectar os nós, selecione ambos os nós e vá em Inserir>conectar nós com seta
- para que a seta fique com boa aparência, o mais fácil é clicar com o botão direito nela e no tipo de seta escolher a opção em que está escrito "simular borda" (ou aresta, ou linha. Estou sem o freeplane aqui e não me lembro como é a tradução). Além disso, você pode alterar o tipo de borda em cada nó (tente colocar linear), para que a conexão fique com mehor aparência.

Me diga se conseguiu um bom resultado, por favor.

Você pode usar estilo e templates para facilitar, caso vá criar vários organogramas.

**Comment by enioluis** [comment]
Tentando aqui amigo, ainda não consegui.

**Comment by euu2021** [comment]
Qual parte não está funcionando? Você conseguiu inserir nós flutuantes?

Veja que todos os nós que você for criando no mapa terão que ser inseridos como nós flutuantes, pois, se você inserir os nós da maneira normal, o programa vai  organizá-los horizontalmente, e não verticalmente.

**Comment by enioluis** [comment]
Este é o problema. Eu insiro os nós flutuantes, mas eles ficam horizontal e não tem jeito maneira de fazer eles virem para baixo

**Comment by euu2021** [comment]
Para mover nós, você deve:

- colocar o mouse do lado esquerdo do nó, de forma que aparecerá uma forma oval.
- aí você clica na forma oval e arrasta para o local desejado

Veja o nó flutuante sendo movido nesse vídeo (no 1 minuto e 20 segundos):
https://youtu.be/EsUyEe_hcsE

**Comment by enioluis** [comment]
Huhuhuhu, deu certo. O problema agora é fazer uma seta reta.

**Comment by euu2021** [comment]
Uma forma de fazer uma seta reta é:

- selecione dois nós e os conecte usando Inserir>conectar nós com seta

- clique com o botão direito na seta e aparecerá uma janela com as configurações da seta

- mude esse opção para algo do tipo linha ou caminho linear:
![IMG_20220618_080606](https://user-images.githubusercontent.com/77707706/174435015-c3f0c11a-51b7-4678-a2bb-e7a68ec4c80a.jpg)

Para não ter que mudar seta por seta, vá no editor de estilos e lá altere as configurações da seta que aparece conectada ao primeiro estilo (que é o estilo padrão ou default).

Eu estou viajando e sem acesso ao Freeplane, senão eu faria um vídeo passo a passo e ficaria mais fácil de entender.
